MySQL_basic_commands/README.md

```diff
@@ -0,0 +1,328 @@
+# Welcome to  Basics of MySQL
+***
+
+## INTRODUCTION
+
+## What is MySQL?
+**MySQL is a standard language for storing, manipulating and retrieving data. It is a relational database managemnet system software.**
+
+## What you will be learning in this Micro-Byte?
+It is really important to get started with the basics and understand the concepts and commands to manipulate data stored in tables in MySQL. MySQL is of primary importance. No matter which field you pursue ahead in your career, be it data science, cloud computing, software development or any related discipline, you will have to know about the fundamentals of DBMS i.e- MySQL. 
+
+### Pre-requisites
+## Level-1
+1.**MySQL Editor**
+I would be using MySQL command line client 8.0. 
+You can download from https://dev.mysql.com/doc/refman/8.0/en/mysql.html .
+After doing the setup. You can access by entering your admin password.The console should look 
+
+![first](https://github.com/happycoder0011/Database/blob/happycoder0011/basics_of_MySQl/add/MySQL_basic_commands/Images/Screenshot%20(83).png)
+
+
+2.**Create Database**
+The tables or the relations are held inside a database. So firstly you will have to create a data base.
+
+Create Database
+```sh
+create database database_name;
+```
+
+
+![second](https://github.com/happycoder0011/Database/blob/happycoder0011/basics_of_MySQl/add/MySQL_basic_commands/Images/Screenshot%20(84).png)
+
+
+**Note:** MySQL is non case sensitive.
```

Comment:
not* - typo

Reply:
I got it. But dont you think giving link for every individual command would be a lot scattered and disrupting the learning. That's why I gave all the content part in the prerequisites. The activities below can be done only if they learn through the content.
The extra bytes(blob, enum). I am adding more references to that. 
Kindly confirm should I remove all the command part. I can add  references to w3school if you say.
Thanks.

Reply:
Sure makes sense. Maybe you can cut down the number of examples by a bit